### PR TITLE
Fork/174 disable update nag

### PR DESCRIPTION
**Pull request by nullvariable**
- Brings file inline with WordPress core coding standards
- Adds `DISABLE_PANTHEON_UPDATE_NOTICES` constant to allow disabling update nags via `wp-config.php`

### CODE REVIEW

**Comment by nullvariable** [comment]
This fixes #174

**Comment by nullvariable** [comment]
@pwtyler this one too :wink:

**Comment by pwtyler** [comment]
Changed the constant from `PANTHEON_UPDATE_NOTICES` to `DISABLE_PANTHEON_UPDATE_NOTICES`, and moved the constant check in `pantheon-login-form-mods.php` up to this file as well for consistency & clarity.

Will update https://github.com/pantheon-systems/documentation/pull/6060 accordingly as well.

cc: @greg-1-anderson & @tylerdigital.

File: wp-content/mu-plugins/pantheon.php [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-require_once( 'pantheon/pantheon-page-cache.php' );
-require_once( 'pantheon/pantheon-updates.php' );
-require_once( 'pantheon/pantheon-login-form-mods.php' );
-
-endif; # Ensuring that this is on Pantheon
+	require_once 'pantheon/pantheon-page-cache.php';
+	if ( ! defined( 'DISABLE_PANTHEON_UPDATE_NOTICES' ) || ! DISABLE_PANTHEON_UPDATE_NOTICES ) {
+		require_once 'pantheon/pantheon-updates.php';
+	}
+	if ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) {
```
Comment (nullvariable) [inline wp-content/mu-plugins/pantheon.php:19]: should this also be a negative condition? ie `! RETURN...`

Comment (nullvariable) [reply wp-content/mu-plugins/pantheon.php:19]: never mind, I get the flow now

**Review by greg-1-anderson — APPROVED** [review]
LGTM

**Review by tylerdigital — APPROVED** [review]

**Pull request merged**